requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
I don't know why this was generated by python 3.7 in the past, but we def run the repo-health job in python 3.8

Reply:
We still haven't dropped `Python 3.7` support from this repo since we thought it may be in use by some open source community and there is still time in the deprecation deadline for `Python 3.7`. That's the reason why the `requirements upgrade job` for this repo has been running with `Python 3.7` instead of `Python 3.8`.